worlds/sc2/__init__.py

```diff
@@ -82,10 +84,12 @@ class SC2World(World):
     locked_locations: List[str]
     """Locations locked to contain specific items, such as victory events or forced resources"""
     location_cache: List[Location]
-    mission_req_table: Dict[SC2Campaign, Dict[str, MissionInfo]] = {}
-    final_mission_id: int
-    victory_item: str
+    # mission_req_table: Dict[SC2Campaign, Dict[str, MissionInfo]] = {}
+    # final_mission_id: int
+    # victory_item: str
```

Comment:
I recommend removing commented out code like this. We can see the old version in the PR, and we don't want it lingering around after a merge if it's not going to be used.

Reply:
Force of habit - I'd rather have working code nearby as a reference than outright remove it. I usually delete these things for PRs, but since it's only a draft I didn't worry about it.